javav2/example_code/ec2/src/test/java/EC2Test.java

```diff
@@ -118,239 +107,159 @@ public static void setUp() throws IOException {
     @Test
     @Tag("IntegrationTest")
     @Order(1)
-    public void CreateInstance() {
-        instanceId = CreateInstance.createEC2Instance(ec2, instanceName, ami);
-        assertFalse(instanceId.isEmpty());
-        System.out.println("\n Test 1 passed");
+    public void createKeyPair() {

```

Comment:
This test isn't testing anything. If `createKeyPairAsync` failed, the test would still pass. This behavior exists in most of the tests.

Reply:
TEsts have been improved and use Assertions, etc.

Reply:
Have the changes been pushed? Please don't mark as resolved until the changes are up.